loadtest/control/simulcontroller/controller.go

```diff
@@ -54,6 +55,9 @@ func New(id int, user user.User, config *Config, status chan<- control.UserStatu
 	}, nil
 }
 
+// All requests are based in API v4, so the absolute minimum version required is 4.0.0
+var initialVersion = semver.MustParse("4.0.0")
```

Comment:
Not very strong opinions on this but I am not sure it would make a lot of sense to support anything prior to v5. This repository was created around 5.17 time and I don't believe it was ever executed against previous versions.

Reply:
Right, and if we are not going to test again a certain version. I think it's just better to restrict.